### DISCUSSION TITLE
Reversing streak loss after accumulating a big enough streak

### DISCUSSION THREAD

**Discussion by snalgae** [ppy/osu#29189] [Ideas] [answered]
In Puzzmo (which is a web-based collection of puzzles (think crossword)) there are daily streaks for completing puzzles. But maintaining streaks is treacherous and can become disheartening when broken. In Puzzmo if you lose a streak and then make a 7-day streak you get back the maximum-length streak you have. Example: You get a 16-day streak then break it, 3-day streak then break it and a 7-day streak, 16 days will be added to the current streak so it will be 23 now (16+7). I know there is a weekly streak to not frustrate players as much, but I'm curious about what are the developers' thoughts about this idea.

**Comment by bdach** [comment, marked as answer]
Don't see the point given the main number on profile is already individual completions and not streaks.

This could lead to situations where your streak is longer than your actual completion count.

**Reply by snalgae** [reply]
oh i guess i thought that was the streak number xd

**Reply by snalgae** [reply]
![Screenshot 2024-07-29 231047](https://github.com/user-attachments/assets/715029ac-f5ed-4b7a-ae1f-975dc94203ec)
uhh this says streak im confused

**Reply by bdach** [reply]
The number next to "daily challenge" is not the same number as "current / best daily streak". That number is not repeated in the tooltip. They just happen to be equal here because... well, that's what the reality is.
